### PR TITLE
Add secureVarsFile to show command

### PR DESCRIPTION
**Pull request by favoretti**
Adding secureVarsFile to `show` command.

### CODE REVIEW

**Review by charleszipp — CHANGES_REQUESTED** [review]
Changes look good to me. Could we get a test scenario added for show that ensures that whats provided to secureVarsFile is used appropriately? This would update environment vars if its a .env. Otherwise should append a `-var-file=` arg. This type of test exist on other commands such as plan/apply/destroy. Should be lots of examples to copy from.

**Comment by favoretti** [comment]
show doesn't accept -var-file, so I need to somehow make sure it's only .env that's supported..

**Comment by charleszipp** [comment]
> show doesn't accept -var-file, so I need to somehow make sure it's only .env that's supported..

It should be possible to validate that .env was provided within the command handler. This is also currently done for the state file input to make sure it has whats expected. Then should be able to validate the negative case where someone tries to provide a tfvars or something other than .env file. Checking the extension I think should be sufficient to start.

**Comment by favoretti** [comment]
@charleszipp Found that I already built in that check for `init`, so added `show` to the same verification clause. Should be good now, if you think the test is ok.

File: TerraformCLI/src/terraform-runner.ts [hunk 1]
```diff
@@ -121,8 +121,8 @@ export class TerraformWithSecureVarFile extends TerraformCommandDecorator{
                     throw "The .env file doesn't have valid entries.";
                 }
             } else {
-                if(context.command.name === 'init') {
-                    throw "terraform init command supports only env files, no tfvars are allowed during this stage.";
+                if(context.command.name === 'init' || context.command.name === 'show') {
+                    throw "terraform "+context.command.name+" command supports only env files, no tfvars are allowed during this stage.";
```
Comment (charleszipp) [inline TerraformCLI/src/terraform-runner.ts:125]: nice! love the surgical approach here ;)

**Review by charleszipp — CHANGES_REQUESTED** [review]
Test looks good. Can we get one to cover the negative case? Should be good after that. Init has a scenario to cover the negative case so should be easy to copy this into show as well. Will help document that show should not allow tfvars files
image

**Comment by favoretti** [comment]
Pushed.

**Review by charleszipp — APPROVED** [review]
Thanks! LGTM

**Pull request merged**